README.md

```diff
@@ -51,14 +51,14 @@ Custom property | Description | Default
 `--ecp-header-height` | Header height | `48px`
 `--ecp-header-bg` | Header background color | `#0099ff`
 `--epc-header-color` | Header color | `#ffffff`
-`--epc-header` | Mixin applied to header | `{}`
+`etools-content-panel::part(epc-header)` | Mixin applied to header | `{}`
```

Comment:
Let's fix the naming also, `epc` versus `ecp`  prefix.  Let's stick to **ecp** as an acronym for **e**tools-**c**ontent-**p**anel

Reply:
Done